src/gui/AstroCalcDialog.hpp

```diff
@@ -104,16 +104,18 @@ class AstroCalcDialog : public StelDialog
 		EphemerisCount          //! total number of columns
 	};
 
-	//! Defines the number and the order of the columns in the transit table
-	//! @enum TransitColumns
-	enum TransitColumns {
-		TransitCOName,          //! name of celestial object
-		TransitDate,            //! date and time of transit
-		TransitAltitude,        //! altitude
-		TransitMagnitude,       //! magnitude
-		TransitElongation,      //! elongation (from the Sun)
-		TransitAngularDistance, //! angular distance (from the Moon)
-		TransitCount            //! total number of columns
+	//! Defines the number and the order of the columns in the rises, transits and sets table
+	//! @enum RTSColumns
+	enum RTSColumns {
+		RTSCOName,		//! name of celestial object
```

Comment:
Again just cosmetics. Better use space, not tab, for aligning the comments.

Reply:
Fixed